src/main/java/com/guysfromusa/carsgame/v1/CarResource.java

```diff
@@ -54,28 +57,31 @@ public Car addCar(@PathVariable("name") String name, @RequestBody String type){
         return toCar(addedCar);
     }
 
-    @PostMapping(path = "{name}/game/{game}")
+    @DeleteMapping(path= "{name}")
+    @ApiOperation(value = "Remove car", notes = "Operation removes car and returns id of deleted car", response = Long.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 204, message = "Car successfully removed")
+    })
+    public Long removeCar(@PathVariable("name") String name){
+        return carService.deleteCarByName(name);
+    }
+
+
+    @PostMapping(path = "{car}/cars/{game}")
     @ApiOperation(value = "Add car to given game", response = Car.class)
     @ApiResponses(value = {
             @ApiResponse(code = 200, message = "Car successfully added to game"),
             @ApiResponse(code = 404, message = "Game not found"),
-            @ApiResponse(code = 404, message = "Car not found")
+            @ApiResponse(code = 404, message = "Car not found"),
+            @ApiResponse(code = 404, message = CAR_CRASHED_MESSAGE),
```

Comment:
For validation result please use 400 Bad request, I think this can be also applied to game not found and car not found as well